app/client/components/liveChat/liveChat.tsx

```diff
@@ -0,0 +1,279 @@
+    color: ${brand[400]};
+  }
+
+  .embeddedServiceSidebarMinimizedDefaultUI .embeddedServiceIcon {
+    display: none;
+  }
+
+  .embeddedServiceSidebarMinimizedDefaultUI .minimizedText > .message {
+    margin-bottom: ${space[1]}px;
+    overflow: visible;
+  }
+
+  .embeddedServiceSidebarMinimizedDefaultUI .minimizedText > .message::before {
+    content: url(${minimisedChatSpeechBubble});
+    position: relative;
+    top: 6px;
+    margin-right: ${space[1]}px;
+  }
+
+  /* Waiting to chat */
+  .embeddedServiceLiveAgentStateWaiting .waitingStateContent {
+    text-align: initial;
+    justify-content: flex-start;
+  }
+
+  .embeddedServiceLiveAgentStateWaiting .waitingMessage {
+    padding: 0;
+  }
+
+  .embeddedServiceLiveAgentStateWaiting .waitingGreetingContent {
+    color: ${neutral[46]};
+    margin: ${space[5]}px 0;
+  }
+
+  .embeddedServiceLiveAgentStateWaiting .waitingGreeting {
+    font-size: 17px;
+  }
+
+  .embeddedServiceLiveAgentStateWaiting .embeddedServiceLoadingBalls {
+    padding-top: 0;
+    align-self: flex-start;
+  }
+
+  .embeddedServiceLiveAgentStateWaiting .loadingBall,
+  .embeddedServiceLoadingBalls.tiny .loadingBall {
+    background-color: #c4c4c4;
+  }
+
+  .embeddedServiceSidebarButton {
+    background: ${brand[400]};
+    border: 1px solid ${neutral[46]};
+  }
+
+  /* Chat header */
+  h2[embeddedService-chatHeader_chatHeader] {
+    overflow: visible;
+  }
+
+  h2[embeddedService-chatHeader_chatHeader]::before {
+    content: url(${chatHeaderIcon});
+    position: relative;
+    top: ${space[2]}px;
+    margin-right: ${space[2]}px;
+  }
+
+  /* Chat body */
+  .embeddedServiceSidebar .sidebarBody {
+    font-family: ${sans};
+  }
+
+  .embeddedServiceSidebarButton:focus {
+    text-decoration: none;
+  }
+
+  /* Chat messages */
+  .embeddedServiceLiveAgentStateChatPlaintextMessageDefaultUI.agent.plaintextContent {
+    background: #ededed;
+  }
+
+  .embeddedServiceLiveAgentStateChatPlaintextMessageDefaultUI.chasitor.plaintextContent {
+    background: ${brand[400]};
+  }
+`;
+
+export const WithLiveChatContainer = ({ children }: PropsWithChildren<{}>) => {
+  const liveChatContainerRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const queryString = window.location.search.slice(1);
+
+    const liveChatRegex = /liveChat.+?(?=\&|$)/g;
+    const match = queryString.match(liveChatRegex);
+
+    if (match) {
+      const liveChatKeyValueArray = match[0].split("=");
+      window.sessionStorage.setItem(
+        liveChatKeyValueArray[0],
+        liveChatKeyValueArray[1]
+      );
+    }
+
+    if (
+      window.sessionStorage.getItem("liveChat") === "1" &&
+      liveChatContainerRef.current
+    ) {
+      initLiveChat(liveChatContainerRef.current);
+    }
+  }, []);
+
+  return (
+    <>
+      {children}
+      <div ref={liveChatContainerRef} css={withLiveChatContainerCss} />
+    </>
+  );
+};
+
+export const StartLiveChatButton = () => {
```

Comment:
Suggestion: Adding `StartLiveChatButton` to the output of `WithLiveChatContainer` and making the the button it returns float at the bottom of the page. This would allow us to merge this PR now and add changes later. It would also allows us to continue testing without developer intervention.